src/OpenLoco/Widget.cpp

```diff
@@ -304,16 +303,16 @@ namespace OpenLoco::Ui
         Gfx::Context* clipped = nullptr;
         if (Gfx::clipContext(&clipped, context, left + window->x, top + window->y, right - left, 41))
         {
-            uint32_t image;
+            uint32_t frame = image;
             if (window->flags & WindowFlags::flag_11)
             {
-                image = 0x20000000 | 2322 | ((colour & 0x7F) << 19);
+                frame = 0x20000000 | 2322 | ((colour & 0x7F) << 19);
```

Comment:
There is a function that could be used for this instead to clear it up.
```imageId = Gfx::recolour(2322, Colour::opaque(colour))```

Reply:
Image 2322 is the frame's background image (800px wide). Please add a constant to Graphics/ImageIds.h for it. (Note that the file is ordered sequentially.)

Reply:
Added constant, named frame_background_image in right order.